lib/commands/smapi/smapi-commander.js

```diff
@@ -20,7 +20,9 @@ const defaultOptions = [{
     description: optionModel.debug.description
 }];
 const requiredTemplate = { true: '[REQUIRED]', false: '[OPTIONAL]' };
-const jsonTemplate = { true: '[JSON]\n', false: '' };
+const jsonTemplate = { true: '[JSON]: Json string or a file. To pass as a file, prefix the file path with "file:". '
++ 'For example: --manifest file:/folder/file.json.\n',
```

Comment:
1. I think there is a need to add usage for different [tag] at the top of smapi command doc.

2. [JSON]: JSON string or a file. Example: "$(cat {filePath})" or "file:{filePath}", either absolute or relative path supported.